### ISSUE TITLE
Security Level

### ISSUE THREAD

**Issue by lelikan-code** [ge0rg/aprsdroid#377]
I couldn't find any documentation regarding the security of the application. Is there any encryption protocol applied to data sent from person to person?

**Comment by lelikan-code** [comment]
What I am curious about is not a general broadcast. After all, if I am only going to talk to one person, I would want it to be sent only to them.

**Comment by penguin359** [comment]
Yes, that is exactly what I am talking about, point-to-point communications with an entity. However, there is no specification for any kind of encryption in APRS or AX.25 which APRS uses for it's transport. Even if there was, I would not be able to test it as it's illegal to use in the United States and in the United Kingdom, the latter is home to the original author. If you are in a country that allows encryption, you are free to come up with your own specification for encryption, fork APRSdroid, and add support for it.

**Comment by na7q** [comment]
> What I am curious about is not a general broadcast. After all, if I am only going to talk to one person, I would want it to be sent only to them.

You're in the wrong place. You should be looking at meshtastic, not ham radio. Based on your statements, I assume you're not a license ham operator either.

**Comment by lelikan-code** [comment]
I assume you are not using the latest versions of VARA HF. The AES256 encryption option is available there. :)

**Comment by penguin359** [comment]
Yes, I actually am aware of that feature in VARA HF and it is licensed for non-HAM use only. In fact, when using a HAM callsign, it is disabled. From what you are describing, it sounds like Meshtastic in non-HAM mode would suit your use case far better.

Out of curiosity, what country will you be operating in with this?
